sp/src/game/shared/SoundEmitterSystem.cpp

```diff
@@ -1001,7 +1001,11 @@ class CSoundEmitterSystem : public CBaseGameSystem
 
 			if ( duration )
 			{
-				*duration = enginesound->GetSoundDuration( pSample );
+				if ( Q_stristr( pSample, ".mp3" ) ) {
```

Comment:
Like the commit message states, this crashes on Linux. I guess it doesn't on Windows, but doesn't return reliable values, taking the following comment from https://www.moddb.com/mods/sourceworld (scripts/vscript/music_manager.nut):
```squirrel
// The price for not using WAV format for looped music, ladies and gents. Manually writing all durations because the engine can't read duration of mp3s correctly.
::SW_MUS<-{}
::SW_MUS.DurTable<-{
	hordeslayer_01=5.6411
	hordeslayer_02=5.6411
	hl2_song1=60
	...
}
```